Integrations/EWSMailSender/EWSMailSender.py

```diff
@@ -259,7 +260,7 @@ def test_module():
 
 
 def main():
-    global USERNAME, PASSWORD, ACCOUNT_EMAIL, log_stream
+    global USERNAME, PASSWORD, ACCOUNT_EMAIL, log_stream, config
```

Comment:
We define config as global also on line 275. I think you can remove it from there.

Reply:
I've removed it from line 275 so it will refer from the `main` function and not only from the `try` block.